frontend/src/mixins/Permissions.js

```diff
@@ -29,7 +29,7 @@ export default {
         },
         /**
          *
-         * @param {'dashboard', 'viewer', 'member', 'owner'} role
+         * @param {Roles} role
```

Comment:
`Roles` (the type) is an enum (look up object). `role` (the variable) is a `role` but seeing as `role` was never `typedef`ed, it falls to being a `number`. 

Either add a typedef for `role` (ideally in the `roles.js` file) and update the function jsdoc

```
/**
 * @typedef {0 | 5 | 10 | 30 | 50 | 99} Role
 * Enum for roles with specific numeric values.
 */
 
/**
 * Check if the user has the minimum required role.
 * @param {Role} role - The role to check against.
 * @returns {boolean} True if the user has the minimum required role, otherwise false.
 */
function hasAMinimumTeamRoleOf (role) {
   ...
}
```

Or simply hard code them

/**
 * Check if the user has the minimum required role.
 * @param {0 | 5 | 10 | 30 | 50 | 99} role - The role to check against e.g `Roles.Member`
 * @returns {boolean} True if the user has the minimum required role, otherwise false.
 * @example
 * // Check if the user has at least the 'Member' role
 * const isMemberOrHigher = hasAMinimumTeamRoleOf(Roles.Member)
 */
function hasAMinimumTeamRoleOf (role) {
   ...
}
```